api/arc.go

```diff
@@ -1416,26 +1476,61 @@ func ParsePOSTTransactionResponse(rsp *http.Response) (*POSTTransactionResponse,
 		}
 		response.JSON400 = &dest
 
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 409:
+		var dest ErrorGeneric
+		if err := json.Unmarshal(bodyBytes, &dest); err != nil {
+			return nil, err
+		}
+		response.JSON409 = &dest
+
 	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 422:
 		var dest Error
 		if err := json.Unmarshal(bodyBytes, &dest); err != nil {
 			return nil, err
 		}
 		response.JSON422 = &dest
 
-	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 465:
-		var dest ErrorFee
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 460:
+		var dest ErrorTxFormat
 		if err := json.Unmarshal(bodyBytes, &dest); err != nil {
 			return nil, err
 		}
-		response.JSON465 = &dest
+		response.JSON460 = &dest
+
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 461:
+		var dest ErrorUnlockingScripts
+		if err := json.Unmarshal(bodyBytes, &dest); err != nil {
+			return nil, err
+		}
+		response.JSON461 = &dest
+
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 462:
+		var dest ErrorInputs
+		if err := json.Unmarshal(bodyBytes, &dest); err != nil {
+			return nil, err
+		}
+		response.JSON462 = &dest
 
-	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 466:
-		var dest ErrorConflict
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 463:
+		var dest ErrorMalformed
 		if err := json.Unmarshal(bodyBytes, &dest); err != nil {
 			return nil, err
 		}
-		response.JSON466 = &dest
+		response.JSON463 = &dest
+
+	case strings.Contains(rsp.Header.Get("Content-Type"), "json") && rsp.StatusCode == 464:
```

Comment:
1. why do we need to cover all the status codes, even those obscure not listed in net/http package? 
2. same magic number as in example above. Can you please define what exact status code you are trying to cover?

Reply:
What I basically did is just to add those errors which ARC in fact returns to the API definition to make it complete. The status codes were already there defined here `api/status.go`